ci-operator/config/openshift/cincinnati-graph-data/openshift-cincinnati-graph-data-master.yaml

```diff
@@ -87,6 +89,7 @@ tests:
     exec hack/validate-blocked-edges.py
   container:
     from: content-checks
+  run_if_changed: blocked-edges/*
```

Comment:
also if the script changes, so `run_if_changed: hack/validate-blocked-edges.py|blocked-edges/*` or some such.